packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -35,8 +42,33 @@ const propTypes = {
   })),
 };
 
+const defaultProps = {
+  isScrollable: true,
+};
+
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (frameOptions.iframeAttrs === null
+      || frameOptions.iframeAttrs === undefined) {
+      frameOptions.iframeAttrs = {};
+    }
+
+    if (frameOptions.resizeConfig === null
+      || frameOptions.resizeConfig === undefined) {
+      frameOptions.resizeConfig = {};
+    }
+
+    if (this.props.isScrollable === false) {
+      Object.assign(frameOptions.resizeConfig, { scrolling: false });
+      Object.assign(frameOptions.iframeAttrs, { scrolling: 'no' });
+    } else {
+      Object.assign(frameOptions.resizeConfig, { scrolling: true });
+      Object.assign(frameOptions.iframeAttrs, { scrolling: 'yes' });
+    }
+
```

Comment:
As I try to understand this code: It seems like lines 51-62 are redundant because regardless of whether those attributes are like, we'll always do the logic in lines 64-70. We could probably even further simplify this line as such:

```suggestion
    // Merging the iframe options props
    const frameOptions = {
      ...this.props.options,
      resizeConfig: { scrolling: this.props.isScrollable },
      iframeAttrs: { this.props.isScrollable ? "yes" : "no" } 
    };
```

Reply:
Reverted this code. There will not be code change for this PR.